frontend/src/scenes/experimentation/FeatureFlags.js

```diff
@@ -122,7 +122,7 @@ function _FeatureFlags() {
                     onClick: () => setOpenFeatureFlag(featureFlag),
                 })}
                 size="small"
-                rowClassName={'cursor-pointer ' + rrwebBlockClass}
+                rowClassName={'cursor-pointer ' + 'ph-no-capture'}
```

Comment:
I know you're not changing anything here, but should feature flag names really be obscured?

Reply:
Oh this I feel more strongly about than prop names. A feature flag name gives me insight into the experiments a company is running. Happy to expand more on the main issue.

Reply:
Alright makes sense, in that case I only think it would be useful to black out specifically the feature flag name and keep everything else?